examples/svg/Main.hs

```diff
@@ -55,8 +55,8 @@ viewModel (Model (x,y)) =
   div_ [ ] [
     svg_ [ style_ $ M.fromList [ ("border-style", "solid")
                                , ("height", "700px")
+                               , ("width", "100%")
                                ]
-         , width_ "auto"
```

Comment:
Not sure what was the intention behind `width "auto"` but in google chrome it renders across entire screen width, and throws this error in js console:
```bash
svg example : lib.js:1811 Error: <svg> attribute width: Expected length, "auto".
```
Whereas in firefox it shows only narrow, but tall svg element and the view looks broken.